src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.container.ts

```diff
@@ -9,21 +9,23 @@ import { getCollectionItems, getLoading as getLoadingItem } from 'modules/item/s
 import { FETCH_COLLECTIONS_REQUEST, DELETE_COLLECTION_REQUEST } from 'modules/collection/actions'
 import { openModal } from 'modules/modal/actions'
 import { FETCH_ITEMS_REQUEST } from 'modules/item/actions'
+import { getThirdParty } from 'modules/thirdParty/selectors'
+import { getThirdPartyId } from 'lib/urn'
 import { MapStateProps, MapDispatchProps, MapDispatch } from './ThirdPartyCollectionDetailPage.types'
 import CollectionDetailPage from './ThirdPartyCollectionDetailPage'
 
 const mapState = (state: RootState): MapStateProps => {
   const collectionId = getThirdPartyCollectionId(state) || ''
   const collection = getCollection(state, collectionId)
-
   return {
     wallet: getWallet(state)!,
     collection,
     items: getCollectionItems(state, collectionId),
     isLoading:
       isLoadingType(getLoadingCollection(state), FETCH_COLLECTIONS_REQUEST) ||
       isLoadingType(getLoadingCollection(state), DELETE_COLLECTION_REQUEST) ||
-      isLoadingType(getLoadingItem(state), FETCH_ITEMS_REQUEST)
+      isLoadingType(getLoadingItem(state), FETCH_ITEMS_REQUEST),
+    thirdParty: collection && collection.urn ? getThirdParty(state, getThirdPartyId(collection.urn)) : null
```

Comment:
We usually try to keep selectors that do a direct access without a method. Still this is not necessarily bad thing, but wanted to add the comment to consider [this](https://github.com/decentraland/builder/pull/1664/files#diff-ee2a3e5bcc9f9f86387798ee5c181abbcbbdd0f52fa80a3c7a71983cbce70bfdR103)

Reply:
I'm not entirely sure I understand this comment. Are you saying that `getThirdParty` shouldn't exist?